### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Tuesday, 14th of March 2023 17:30 CET / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#973] [Flatcar Office Hours]
## Agenda for the Flatcar Office Hours on Tuesday, 14th of March 2023 17:30 CET / 11:30am EDT / 8:30am PDT

- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording: [here](https://www.youtube.com/watch?v=acAFlx2fGiE)
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  
## Welcome
- Brief intro/check-in of the new participants joining the call. Please introduce yourself and share what brings you here today.

## Agenda items
- Flatcar ClusterAPI OpenStack provider demo / walk-through
- Call out Documentation Writing Day on Friday, 17th of March

### Release Planning
- https://github.com/orgs/flatcar/projects/7/views/8

**Comment by sayanchowdhury** [comment]
I would like to propose to do the Documentation Office Hours on the 17th of March. The whole core team would be there during that week.

**Reply by tormath1** [reply]
Good idea. We could focus on Butane / ct documentation with the idea of keeping one reference to `ct` for LTS releases (which still use Ignition v2) and Butane for the rest.

**Reply by tormath1** [reply]
We can write some documentation for flatcar OS reset tool and `/etc` being an overlayfs (with cool tricks like: `sudo git diff --no-index /usr/share/flatcar/etc /etc`).

EDIT: There is now a PR https://github.com/flatcar/flatcar-docs/pull/279

**Reply by pothos** [reply]
For the CLC and Butane separation I prepared https://github.com/flatcar/flatcar-docs/pull/280 - we don't need to remove the CLC config docs as we continue to support them.
What we should do, though, is to migrate all the examples like https://www.flatcar.org/docs/latest/setup/clusters/architectures/#configuring-your-laptop to Butane, in particular the removed sections like `networkd:` and the other ones mentioned in https://github.com/flatcar/Flatcar/issues/968#issuecomment-1465834953.

**Comment by tormath1** [comment]
If nothing else is proposed, I can present a quick demo of Flatcar + Cluster API OpenStack. Regarding the recent merging of https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1444

**Reply by tormath1** [reply]
@lentzi90 and @mdbooth you are of course invited to join :)

**Reply by mdbooth** [reply]
I'll be interested to see that! I've added it to my calendar and I'll attend if I can.
